app/editor/src/features/admin/filters/FilterSettingsForm.tsx

```diff
@@ -65,10 +65,11 @@ export const FilterSettingsForm: React.FC<IFilterSEttingsFormProps> = ({
       var values = { ...settings };
       values[key] = value;
       if (key === 'dateOffset') {
-        values = { ...settings, startDate: undefined, endDate: undefined };
+        values = { ...values, startDate: undefined, endDate: undefined };
       } else if (key === 'startDate' || key === 'endDate') {
-        values = { ...settings, dateOffset: undefined };
+        values = { ...values, dateOffset: undefined };
       }
+      values = { ...values, validDateOffset: key === 'dateOffset' && value === 3 };
```

Comment:
is the purpose of `validDateOffset` to determine if the date filter is a range vs an offset?  Seems a bit funny that we have two values `dateOffset` and `validDateOffset` which seem to overlap functionality

Reply:
No, it is not for that, in this case it is used for telling if we should use the value 3 or not as in some case that should represent ALL instead, it could be a bit confusing as naming is hard sometimes :)

Reply:
It's very confusing.  The Editor app uses the following values.
0 = today
1 = 24 hours
2 = 48 hours
3 = all
What values is the Subscriber app using?  We should be consistent.

Reply:
In Editor app we use 3 for either ALL or 3 in different places such as the home search and the admin/filters Query, so validDateOffset is here for telling the difference between the two, and we can rename it to whatever that make senses.

Reply:
Standardize the values.
 
Perhaps the following makes more sense
0 = All
1 = Today
2 = 24 hours
3 = 48 hours

Reply:
Remove this logic on this line.

Reply:
Removed, however this rollback has caused a [Search ALL] problem on the home page, which should have now been resolved in this PR as well. Thanks